tests/pytests/unit/modules/test_nacl.py

```diff
@@ -1,10 +1,49 @@
 """
     Unit tests for the salt.modules.nacl module
 """
+import pytest
 
-import salt.modules.nacl
+import salt.utils.stringutils
 from tests.support.mock import patch
 
+try:
+    import salt.modules.nacl as nacl
+
+    HAS_PYNACL = True
+except (ImportError, OSError, AttributeError):
+    HAS_PYNACL = False
+
+
+@pytest.mark.skipif(
+    not HAS_PYNACL, reason="skipping test_nacl, reason=PyNaCl is unavailable"
+)
```

Comment:
```suggestion
import salt.modules.nacl as nacl

pytest.importorskip("libnacl")
```

Reply:
@s0undt3ch I am hoping the 'libnacl' is a typo ?, believe that should be 'pynacl' since we are eliminating use of 'libnacl'.

Reply:
Updated suggestion.